sprint_statistics.rb

```diff
@@ -2,33 +2,12 @@
 require 'active_support/core_ext'
 
 class SprintStatistics
-  def initialize(access_token, milestone_string = nil)
```

Comment:
Other scripts are still using this `milestone_string`

Reply:
Should I update the callers in this PR or in followup PRs?  I am not sure if they even work.

Reply:
Changes in this file aren't necessary for the rest of the PR, right?

Reply:
No.  Will remove the unrelated changes to remove milestone.